js/components/accessibleButton.js

```diff
@@ -108,14 +109,19 @@ AccessibleButton.propTypes = {
 
 // Define focusId as a getter so that it returns a different value
 // for each instance of AccessibleButton (defaultProps is static)
-AccessibleButton.defaultProps = Object.create({}, {
-  focusId: {
-    enumerable: true,
-    get: function() {
-      return Math.random().toString(36).substr(2, 10);
+AccessibleButton.defaultProps = Object.create(
+  {},
+  {
+    focusId: {
+      enumerable: true,
+      get: function() {
+        return Math.random()
```

Comment:
I think this fix is goingto create issues if it auto inserts a ';'.  We should change this to store it in a local variable and return that. Maybe the autoformatter won't change it then.

Reply:
\+